hazelcast/src/test/java/com/hazelcast/nio/tcp/nonblocking/iobalancer/IOBalancerStressTest.java

```diff
@@ -112,7 +112,7 @@ public String debug(TcpIpConnectionManager connectionManager) {
             sb.append(in + " :" + in.getEventCount() + "\n");
 
             for (TcpIpConnection connection : connectionManager.getActiveConnections()) {
-                NonBlockingReadHandler readHandler = (NonBlockingReadHandler) connection.getReadHandler();
+                NonBlockingSocketReader readHandler = (NonBlockingSocketReader) connection.getSocketReader();
```

Comment:
Should be the last `NonBlockingSocketReader readHandler` and `NonBlockingSocketWriter writeHandler` pair.